File: hw/vargas/u3/hw28PrinciplesAplication/python/DB_Connection.py
from pymongo import MongoClient

class DB_Connection:
    _URI = "mongodb+srv://Cesar:Cesar2006@cluster0.tgbv2qc.mongodb.net/"
    _DB_NAME = "ResourcesDB"
    
    @staticmethod
    def get_database():
        try:
            client = MongoClient(DB_Connection._URI)
            return client[DB_Connection._DB_NAME]
        except Exception as e:
            print(f"Connection Error: {e}")
            return None

File: hw/carvajal/u3/hw28PrinciplesAplication/principles/src/main/java/ec/edu/espe/storemanagement/utils/DBConnection.java
package ec.edu.espe.storemanagement.utils;

/**
 *
 * @author Adrian Toapanta, @ESPE
 */

import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoDatabase;

public class DBConnection {
    private static final String URI = "mongodb+srv://Emily:Emily2006@cluster0.ynnit6l.mongodb.net/";
    private static MongoClient mongoClient = null;

    public static MongoDatabase getDatabase() {
        if (mongoClient == null) {
            mongoClient = MongoClients.create(URI);
        }
        return mongoClient.getDatabase("StoreDB");
    }
}
